packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -14,17 +14,7 @@
   cursor: not-allowed;
 }
 
-// Fix to off-centered content in V3 when icon present
-.va-button--icon {
-  position: relative;
-  top: 0.1em;
-  &.margin-right-8px {
-    margin-right: 8px;
-  }
-  &.margin-left-8px {
-    margin-left: 8px;
-  }
-}
```

Comment:
With [icon in button support](https://github.com/uswds/uswds/pull/5398/files) added in USWDS v3.8.0, we no longer need these styles.